research-app-messages/src/selections.ts

```diff
@@ -1,3 +1,57 @@
+export enum ImageSetType {
```

Comment:
Hmmm. I think it would be good not to define ImageSetType as an enum here. The reason is that a big purpose of the research-app-messages module is to make it clear how the messages are expressed in a JSON serialization, and the use of an enum type kind of obscures how that works. In practice, we would have that `ImageSetInfo.type` is a number with limited allowed values (I think...), and it's going to be hard to remember what the various magic numbers mean when reading the raw JSON.

The way that enums are handled in other parts of this module (e.g. `classic_pywwt.CreateAnnotationMessage.shape`) is to use a TypeScript literal union type:

```
shape: "circle" | "line" | "polygon";
```

This makes the JSON representation much clearer to the reader, I think, and also results in more readable JSON data. It's a minor hassle to translate the TypeScript enum to this expression but I think it would be worth it here.